lib/customs.js

```diff
@@ -52,6 +52,9 @@ module.exports = function (log, error) {
           }
           throw error.requestBlocked()
         }
+        if (result.suspect) {
+          request.app.isSuspiciousRequest = true
+        }
```

Comment:
The alternative seems to be to throw an error, catch it immediately after the call to check, look for the suspicious request error, re-throw the rest, set request.app.isSuspiciousRequest or pass a new parameter to shouldEnableSigninConfirmation, continue. That might be OTT.